### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by brownd1978**
Speed up Extracted CRV digitization.  More script standardization.

### CODE REVIEW

File: JobConfig/digitize/Extracted.fcl [hunk 1]
```diff
@@ -29,4 +29,6 @@ physics.producers.CrvCoincidence.PEthresholds : [  8 ,  8 ,  8 ]
 physics.producers.CrvCoincidence.adjacentPulseTimeDifferences  : [ 10 , 10 , 10 ]
 physics.producers.CrvCoincidence.maxTimeDifferences            : [ 10 , 10 , 10 ]
 physics.producers.CrvCoincidence.coincidenceLayers             : [  3 ,  3 ,  3 ]
```
Comment (ehrlich-uva) [inline JobConfig/digitize/Extracted.fcl:31]: CrvCoincidence is an outdated module. It is superseded by CrvCoincidenceClusterFinder.
Shouldn't the settings for CrvPhotons and CrvCoincidenceClusterFinder come from Offline/CRVResponse/fcl/epilog_extracted.fcl?

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Comment by kutschke** [comment]
@ehrlich-uva If you think this is ready to go, please click the approve button so that we keep the paperwork straight,

File: JobConfig/digitize/Extracted.fcl [hunk 2]
```diff
@@ -24,9 +24,5 @@ physics.producers.CrvPhotons.lookupTableFileNames : ["CRVConditions/v6_0/LookupT
 # for 68 PE/SiPM @ 1 m away from SiPM (Test beam June 2017)
 # using the pulse height calibration, this value gives 45 PE/SiPM @ 1 m away from SiPM
 physics.producers.CrvPhotons.scintillationYields  : [39400,39400,39400]
```
Comment (ehrlich-uva) [inline JobConfig/digitize/Extracted.fcl:26]: I'm not sure, if the CrvPhotons settings are needed here. They are already in Offline/CRVResponse/fcl/epilog_extracted.fcl - except the scintillationYields, which should probably also go to Offline/CRVResponse/fcl/epilog_extracted.fcl.

**Review by ehrlich-uva — APPROVED** [review]

**Comment by kutschke** [comment]
@FNALbuild run build test

**Pull request merged**